x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ccr/action/PutFollowAction.java

```diff
@@ -121,6 +124,7 @@ public static Request fromXContent(final XContentParser parser, final String fol
 
         private String remoteCluster;
         private String leaderIndex;
+        private boolean waitForCompletion;
```

Comment:
This parameter needs to be documented.

Reply:
Also the `PutFollowRequest` class needs to be updated in HLRC. This can be done in a follow up.